src/main/resources/templates/textquestion.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Text Question</title>
+</head>
+<body>
+<form>
+    <div th:fragment="text">
+        <div th:text="${question.getDescription()}"></div>
+        <input type="text" th:maxlength="${question.getCharLimit()}"/>
```

Comment:
Would it be better to use a text area input here?

Reply:
Done! Changed to textarea element